Dipti/Project-CMA/src/index.css

```diff
@@ -0,0 +1,116 @@
+/* Global */
+
+* {
+  font-family: Avenir, -apple-system, BlinkMacSystemFont, Segoe UI, Roboto,
+    Helvetica Neue, Arial, Noto Sans, sans-serif, Apple Color Emoji,
+    Segoe UI Emoji, Segoe UI Symbol, Noto Color Emoji;
+}
+
+/* Chats Page */
+
+.chats-page {
+  position: absolute;
+  top: 0px;
+  left: 0px;
+  width: 100vw;
+  height: 100vh;
+  /* background-color: #545ef4; */
+}
+
+
+.nav-bar {
+  width: 100%;
+  height: 66px;
+  /* background-color: #002766; */
```

Comment:
Either remove the unwanted comments or explain why this is being kept as a comment